tensorflow/python/ops/image_ops_test.py

```diff
@@ -1047,6 +1047,46 @@ def testRandomFlipLeftRight(self):
       self.assertGreaterEqual(count_flipped, 20)
       self.assertGreaterEqual(count_unflipped, 20)
 
+  def testRandomFlipLeftRightWithBatch(self):
+    batch_size = 16
+    seed = 42
```

Comment:
Can you adjust this test so that it uses the seed? It looks like the other test above also doesn't use it. Please fix that as well while you're at it.